static/js/components/email/lib.js

```diff
@@ -115,9 +118,24 @@ export const LEARNER_EMAIL_CONFIG: EmailConfig = {
 
   getEmailSendFunction: () => sendLearnerMail,
 
+  useSendEmail: true,
   emailSendParams: (emailState) => ([
     emailState.inputs.subject || '',
     emailState.inputs.body || '',
     emailState.params.studentId
   ])
 };
+
+export const convertEmailEdit = editObj(([k, v]) => (
+  [k.match(/^subject$|^body$/) ? `email_${k}` : k.replace(/^email_/, ""), v]
+));
+
+export const AUTOMATIC_EMAIL_ADMIN_CONFIG: EmailConfig = {
+  title: 'Edit Message',
+  getEmailSendFunction: () => actions.automaticEmails.patch,
```

Comment:
I'm a bit wary of overloading the meaning of `getEmailSendFunction` to patch the email instead. Can we make a separate function which does the edit and make it a no-op for the other three configs?